scripts/variantstore/variant_annotations_table/README.md

```diff
@@ -0,0 +1,111 @@
+# Creating the Variant Annotations Table
+
+### The VAT pipeline is a set of WDLs
+- [GvsCreateVAT.wdl](scripts/variantstore/wdl/GvsCreateVAT.wdl)
+- [GvsValidateVAT.wdl](scripts/variantstore/variant_annotations_table/GvsValidateVAT.wdl)
+
+The pipeline takes in a jointVCF and outputs a table in BigQuery.
+
+**GvsCreateVAT** creates the table, while
+**GvsValidateVAT** checks and validates the VAT
+
+
+### Run GvsCreateVAT:
+
+Most of the inputs are constants — like the reference, or a table schema — and dont require any additional work (and paths can be found in the [example inputs json](scripts/variantstore/wdl/GvsCreateVAT.example.inputs.json)). However for the specific data being put in the VAT, three inputs need to be created.
+The first two of these inputs are two files — one of the file/vcf/shards you want to use for the VAT, and their corresponding index files. These are labelled as `inputFileofFileNames` and `inputFileofIndexFileNames`.
+
+You will need to know where all of the GVS extract output files live.
+Example of creating the files of file names and indices:
+`gsutil ls gs://prod-drc-broad/beta-release-99k-v3/beta_99k_*.vcf.gz > inputFileofFileNames.txt`
+`gsutil ls gs://prod-drc-broad/beta-release-99k-v3/beta_99k_*.vcf.gz.tbi > inputFileofIndexFileNames.txt`
+
+Those two created files will then need to get copied into a GCP bucket that this pipeline will have access to (eg. this bucket: `gs://aou-genomics-curation-prod-processing/vat/`)
+
+The third input is the ancestry file that Lee has made and it needs to be copied into a GCP bucket that this pipeline will have access to (eg. this bucket: `gs://aou-genomics-curation-prod-processing/vat/`) for easy access during the workflow. This input has been labelled as the `ancestry_file`.
+Most of the other files are specific to where the VAT will live, like the project_id and dataset_name and the table_suffix which will name the VAT itself as vat_`table_suffix` as well as a GCP bucket location, the output_path, for the intermediary files and the VAT export in tsv form.
+
+The other inputs can be copied from where I’ve put several files (maybe we should find a better place?!? Since many are in my scratch dir for now—tho anything in my scratch dir is also living in github as it’s things like what BQ schema is needed)
+
+The provided [example inputs json](scripts/variantstore/wdl/GvsCreateVAT.example.inputs.json) indicates all of the needed inputs.
```

Comment:
Rather than direct the reader to yet another file, I think the comment about hard-coding or variable-izing (I know, not a word) makes more sense and would improve usability.

Reply:
great--I will hardcode for now and then move the files to a better place in the future. 

I wish WDLs could read from local files more easily